bowling/src/frameParser.cpp

```diff
@@ -0,0 +1,54 @@
+#include "frameParser.hpp"
+
+#include <iostream>
+#include <iterator>
+#include <algorithm>
+
+parsedFrame FrameParser::parse(const std::string& gamePoints) {
+  std::vector<size_t> parsed {};
+  size_t last_parsed_char{};
+  auto tokenized = Tokenizer::splitToFramesAlt(gamePoints);
```

Comment:
hmm,  maybe better name should be  pointList ? parsedList?   what exaclty splitToFramesAlt does?

Reply:
split it into two methods something like: 
```suggestion
  auto playerName = readPlayerName(gameLine);
  auto gameSequence = readSequence(gameLine);
```
you dont need to use pairs